app/settings/__init__.py

```diff
@@ -17,6 +17,8 @@ def get_settings():
         set_settings(secret='super secret key', app_name='Open Event')
     else:
         current_app.config['custom_settings'] = make_dict(s)
+        if 'secret' not in current_app.config['custom_settings'] or not current_app.config['custom_settings']['secret']:
```

Comment:
instead of this, do this

` if current_app.config.get('custom_settings') is not None:`

Reply:
But it will only check if the custom_settings is none or not, I want to check if it contains the secret key or not and if it's null or not.

Reply:
I think he meant `current_app.config['custom_settings'].get('secret') is None`